profile/README.md

```diff
@@ -1 +1,98 @@
-This organization contains various repositories for utilities that are used on physical oceanography cruises.
+# ![](https://avatars.githubusercontent.com/GO-SHIP-Oceanography?size=28) GO-SHIP Oceanography
+
+This organization contains
+[various repositories](https://github.com/orgs/GO-SHIP-Oceanography/repositories)
+for utilities that are used on physical oceanography cruises.
+
+The software provided here is community-developed, free, and open-source.
+It has been collated under the [GO-SHIP](https://www.go-ship.org/) program, but is
+open to anyone to download and use.
+
+
+## Usage
+
+Each [repository](https://github.com/orgs/GO-SHIP-Oceanography/repositories) under
+this organisation represents a single tool for use on cruise.
```

Comment:
I would add a sentence to point out that we will use different coding languages, something like: "Please note that different repositories contain software in different programming languages, such as Python, MatLab or spreadsheets. Whenever possible (and reasonable), the code is provided in all available programming languages."

We also need to decide what to do with codes for the same purpose but written in different languages: do we keep them all in the same repository or not? What's the best practice?

Reply:
I have added a note as you describe under `### Repositories`.

As for how to treat multiple languages, if the code is a short script and both implementations fall under the same license (and authors?) I would probably put them in the same place. This way it is also easier to try and keep updates applying to both languages, not just one.

If it is a bigger code with several files to manage, however, then we might create 2 separate repos and users can choose the appropriate one.

I'd say probably discuss with authors and try to be sensible on a case-by-case basis.